db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java

```diff
@@ -51,6 +51,8 @@
 
 import java.util.*;
 
+import java.util.*;
```

Comment:
A duplicated line.

Reply:
thanks @changli6 , I removed the duplicate lines.